Song.Tests/VerseWriterTests.cs

```diff
@@ -0,0 +1,112 @@
+using System.Security.Cryptography;
+using System;
+using System.Text;
+
+namespace Song.Tests
+{
+    public class VerseWriterTests
```

Comment:
Forgot to rename the file as well.

Reply:
I'm not exactly sure what you meant by this, could you elaborate?